auth/api/iam/api.go

```diff
@@ -59,6 +79,17 @@ func (r Wrapper) Routes(router core.EchoRouter) {
 			return audit.StrictMiddleware(f, vcr.ModuleName+"/v2", operationID)
 		},
 	}))
+	// The following handler is of the OpenID4VCI wallet which is called by the holder (wallet owner)
+	// when accepting an OpenID4VP authorization request.
+	router.POST("/iam/:did/openid4vp_authz_accept", r.handlePresentationRequestAccept, audit.Middleware(vcr.ModuleName+"/v2", "openid4vp_authz_accept"))
+	// The following handler is of the OpenID4VP verifier where the browser will be redirected to by the wallet,
+	// after completing a presentation exchange.
+	router.GET("/iam/:did/openid4vp_completed", r.handlePresentationRequestCompleted, audit.Middleware(vcr.ModuleName+"/v2", "openid4vp_completed"))
+	// The following 2 handlers are used to test/demo the OpenID4VP flow.
+	// - GET renders an HTML page with a form to start the flow.
+	// - POST handles the form submission, initiating the flow.
+	router.GET("/iam/:did/openid4vp_demo", r.handleOpenID4VPDemoLanding)
+	router.POST("/iam/:did/openid4vp_demo", r.handleOpenID4VPDemoSendRequest)
 }
```

Comment:
only 2/4 have audit middleware. Is this correct? (echo router has a [Group](https://echo.labstack.com/docs/routing#group) method. Probably doesn't apply here though)

Reply:
fixed